playbooks/group_vars/all/global.yml

```diff
@@ -1,108 +1,34 @@
 ---
 # run roles based on certain params
+skip_clean: false
 skip_prereqs: false
-force_minishift_install: false
-setup_minishift: true
-start_minishift: true
-force_repo_clone: true
-setup_jenkins: true
-setup_fedmsg: true
-setup_containers: true
+skip_artefacts: false
+skip_minishift: false
+skip_helm: false
```

Comment:
When do you use this in the playbooks?

Reply:
I used extensively in 'when' parameter of all tasks to following the rules of Ansible Lint and permit to skip a all of a specific role.